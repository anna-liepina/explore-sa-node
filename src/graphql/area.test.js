describe.skip('GraphQL: Area', () => {
    describe('query', () => {
        it(`xxx`, async () => {
            const { data } = await query({
                query: `
                {
                }`
            });

            expect(data).toMatchSnapshot();
        });

    });

    describe('mutation', () => {
        it(`xxx`, async () => {
            const { data, errors } = await query({
                query: `
                mutation {
                }`
            });

            expect(data).toMatchSnapshot();
        });
    });
});
