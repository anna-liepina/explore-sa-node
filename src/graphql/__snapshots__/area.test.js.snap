// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GraphQL: Area mutation xxx 1`] = `undefined`;

exports[`GraphQL: Area query xxx 1`] = `undefined`;
